applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/pm_static_ZDebug.yml

```diff
@@ -0,0 +1,41 @@
+app:
+  address: 0x10200
+  region: flash_primary
+  size: 0xe7e00
+mcuboot:
+  address: 0x0
+  region: flash_primary
+  size: 0x10000
+mcuboot_pad:
+  address: 0x10000
+  region: flash_primary
+  size: 0x200
+mcuboot_primary:
+  address: 0x10000
+  orig_span: &id001
+  - mcuboot_pad
+  - app
+  region: flash_primary
+  size: 0xe8000
+  span: *id001
+mcuboot_primary_app:
+  address: 0x10200
+  orig_span: &id002
+  - app
+  region: flash_primary
+  size: 0xe7e00
+  span: *id002
+settings_storage:
+  address: 0xf8000
+  region: flash_primary
+  size: 0x8000
```

Comment:
I think 32k is ok (it is a lot of data, and if more is needed we could still have it on external flash) but @kkasperczyk-no mentioned 64k during the talks (@kkasperczyk-no , will 32k be fine or there is a genuine need you foresee in the future?).

Reply:
I would like to move the static PM configuration and refer to it from all the DFU-compatible samples and applications. I think we should do the same for MCUboot configuration. Otherwise maintenance and adding new samples could cause problems.

Reply:
For now we could keep it here and move it in the future.

Reply:
I understand it but am not big fan of it. Not because the idea is bad but because it may be pain to introduce.
What would be a common thing for it? Can we have partition manager configuration in some random directory? That is something we could discuss later and move (if possible) while working on samples.

Reply:
I think we actually need a common directory at least for used configuration of MCUBoot and Partition Manager. Otherwise keeping these consistent among compatible samples would be really difficult.

Reply:
@pdunaj for now 32k is fine and I really have a hope that it should be enough for us, however I just can't predict what is going to happen as Matter spec is not complete yet, and that's why I said 64k. Will we be able to move some settings to external flash? Probably yes, maybe not all of them, but some part for sure. @Damian-Nordic any thoughts on that?